qutebrowser-profile

```diff
@@ -60,7 +60,7 @@ Released under the MIT Licence. See LICENCE file for full licence terms.
 EOF
 
 set -eu
-QP_VERSION="0.0.1"
+QP_VERSION="0.0.2"
```

Comment:
Remove this - if we want to bump the version we'll do this in a dedicated PR so it's clear.